### PR TITLE
save_hdfeos5.py and smallbaselineApp.py: adding avgSpatialCoh.h5 file to S1*.he5 file

### PR DESCRIPTION
**Pull request by ranneylxr**
**Description of proposed changes**

save_hdfeos5.py and smallbaselineApp.py: adding avgSpatialCoh.h5 file to S1*.he5 file

**Reminders**
- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.

### CODE REVIEW

**Comment by yunjunz** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- mintpy/save_hdfeos5.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/insarlab/MintPy/pullRequest?prid=6122943&bid=20032601)

File: mintpy/save_hdfeos5.py [hunk 1]
```diff
@@ -42,8 +42,10 @@ def create_parser():
     parser.add_argument('timeseries_file', default='timeseries.h5', help='Timeseries file')
     parser.add_argument('-t', '--template', dest='template_file', help='Template file')
 
-    parser.add_argument('-c', '--coherence', dest='coherence_file', required=True, 
-                        help='Coherence/correlation file, i.e. avgSpatialCoh.h5, temporalCoherence.h5')
+    parser.add_argument('-tc', '--temporal-coherence', dest='temporalcoherence_file', required=True, 
+                        help='Temporal coherence/correlation file, i.e. temporalCoherence.h5')
+    parser.add_argument('-asc', '--average-spatial-coherence', dest='avgspatialcoherence_file', required=True,
+                        help='Average spatial coherence file, i.e. avgSpatialCoh.h5')
```
Comment (yunjunz) [inline mintpy/save_hdfeos5.py:48]: All options with more than one character are with two "-" in mintpy, could you adjust these two also?

**Review by yunjunz — CHANGES_REQUESTED** [review]
There is a conflict identified in smallbaselineApp.py, could you resolve it?

**Comment by ranneylxr** [comment]
Hi Yunjun,
Sorry, I just saw these comments. I changed the '-tc' and '-asc' to '--tc' and '--asc', and solve the conflicts about smallbaselineApp.py.
Please check.

**Comment by yunjunz** [comment]
Hi @ranneylxr, no worries at all. Thank you for contributing!

BTW, to update a PR, you can simply keep updating the feature you were working on in your fork, then the attached PR will be updated automatically, no need to close and issue a new PR. Check the [contributing guide](https://github.com/insarlab/MintPy/blob/main/docs/CONTRIBUTING.md#4-review-process) if you are interested.

**Pull request closed without merging**